src/main/java/thaumicenergistics/common/tiles/TileAdvancedInfusionProvider.java

```diff
@@ -139,29 +129,19 @@ public void grabAllAspects() {
     @Override
     @TileEvent(TileEventType.WORLD_NBT_WRITE)
     public void onSaveNBT(final NBTTagCompound data) {
```

Comment:
Do only one of these:
1) Either save coordinates of all bound matrices using `NBTTagList`
2) Or remove this method since it does exactly the same as base one.